contracts/vaults/Claim.sol

```diff
@@ -126,13 +143,16 @@ contract Claim is Base {
         Claim storage claim = claims[_claimId];
         uint256 pid = claim.pid;
         // solhint-disable-next-line not-rely-on-time
-        require(msg.sender == owner() || claim.createdAt + 5 weeks < block.timestamp, "HVE09");
+        require(
+          ((msg.sender == arbitrator && claim.isChallenged) ||
```

Comment:
Not clear from the specs if this shouldn't be allowed to still be called at any time by the owner